napari_sphinx_theme/napari.html

```diff
@@ -0,0 +1,67 @@
+<svg
```

Comment:
What is this file? Is it a tiny svg logo? If so, could it be renamed to indicate that?

Reply:
They are (this and all the other partials too). Because the theme expects templates here, it won't easily accept svg files but it will accept an html. To change that would imply a more complicated change of the footer template. I can take a look if there's no rush here.